packages/langium/src/parser/langium-parser.ts

```diff
@@ -518,9 +519,13 @@ class ChevrotainWrapper extends EmbeddedActionsParser {
     definitionErrors: IParserDefinitionError[];
 
     constructor(tokens: TokenVocabulary, config?: IParserConfig) {
+        const useDefaultLookahead = config && 'maxLookahead' in config;
         super(tokens, {
             ...defaultConfig,
-            ...config
+            ...config,
+            lookaheadStrategy: useDefaultLookahead
+                ? new LLkLookaheadStrategy({ maxLookahead: config.maxLookahead })
+                : new LLStarLookaheadStrategy()
```

Comment:
Can we swap the new property with the `...config` object so we can explicitly override this strategy?